models/src/main/java/com/vimeo/networking2/Album.kt

```diff
@@ -8,112 +10,117 @@ import java.util.*
 /**
  * Group of videos to share publicly or privately.
  */
+@JsonClass(generateAdapter = true)
 data class Album(
 
     /**
      * Hexadecimal color code for the decorative color.
      * For example, album videos use this color for player buttons.
      */
+    @Json(name = "brand_color")
     val brandColor: String? = null,
 
     /**
      * The time in ISO 8601 format that the album was created.
      */
+    @Json(name = "created_time")
     val createdTime: Date? = null,
 
-    /**
-     * Custom logo.
-     */
-    val customLogo: PictureCollection? = null,
```

Comment:
a few properties were removed from this class, was that intentional?

Reply:
I added it back in.